.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "./index.js",
+    "rules": {
+      // disable requiring trailing commas because it might be nice to revert to
+      // being JSON at some point, and I don't want to make big changes now.
+      "comma-dangle": 0,
```

Comment:
should we not be putting these rules in the actual ruleset?

Reply:
Nope, these are purely to allow it to lint itself when we write custom rules and not have to deal with painful test writing.